CONTRIBUTING.md

```diff
@@ -1,32 +1,9 @@
 # Contributing to this project
 
-## Joining the Working Group
-
-We are looking for participation from software developers, localization engineers and others with experience
-in Internationalization (I18N) and Localization (L10N). If you wish to contribute to this work, please review
-the information on the Contributor License Agreement below. In addition, you should:
-
-1. Apply to join our [mailing list](https://groups.google.com/a/chromium.org/forum/#!forum/message-format-wg)
-2. Watch this repository (use the "Watch" button in the upper right corner)
-
-<!-- boilerplate follows - do not edit -->
```

Comment:
@aphillips note the 'do not edit' ^

Reply:
I did. I removed it because I removed the boilerplate there.